### ISSUE TITLE
Accessibility/UI: distinguish internal/external labels in action view without requiring (unsignaled) click

### ISSUE THREAD

**Issue by rahaeli** [bluesky-social/ozone#317]
The "take moderation action" screen shows labels, but doesn't distinguish between labels assigned by natively by your moderation service from labels assigned by Bluesky Moderation Services (or any other labeler your account is subscribed to) except with a click. This is not only annoying UI for non-Bluesky mod services, but is also a violation of WCAG [SC 2.4.4](https://w3.org/TR/WCAG21#link-purpose-in-context), [SC 2.4.6](https://w3.org/TR/WCAG21#headings-and-labels), and [SC 3.3.2](https://w3.org/TR/WCAG21#labels-or-instructions) because there's no indication a click will provide that information or that the bubbles are clickable.

A quick fix for the "not able to distinguish label source" UI problem would be to split that line into two lines, distinguishing the native labels from the external labels: keep the "Labels" header and split the actual labels display into two lines. I suggest it as:

**Internal**: bad-person, kicks-kittens, jerk
**External**: rude, spam

The WCAG violations are harder. The best solution I have that doesn't require a lot of redesign is to add the standard visual vocabulary triangle icon to the bubble, pointing right when unclicked and swapping to pointing down when clicked to indicate it's been unrolled, which adds a tiny bit of visual clutter but pretty much knocks out all three violations if appropriately alt-texted ("Expand"/"Collapse" is pretty standard there) and the hitbox is the entire label, not just the triangle (otherwise it would be a violation of 2.5.5 because the target would be smaller than 44x44px).
